nfv_tempest_plugin/tests/common/shell_utilities.py

```diff
@@ -378,14 +378,17 @@ def run_hypervisor_command_build_from_config(file_path, search_param,
         assert result != '', "{} {}".format(msg, hypervisor)
 
         result = "[" + result.replace('\n', ", ") + "]"
-        dev_names = [x.get('devname') for x in json.loads(result)]
-        # The nic-partitioning deployment does not store "devname" param.
+
```

Comment:
Fine. I will try to have a check and keep both methods. Will update after I run the tests.

Reply:
I don't think we should keep _devname_ (it can be different depending on the Operating System) because in theory we must use _address_ (it doesn't change unless you change slot in the server).

Reply:
Fine. I will leave it as is then.